client/landing/stepper/declarative-flow/internals/steps-repository/site-migration-instructions/steps/use-steps.tsx

```diff
@@ -0,0 +1,132 @@
+import { useTranslate } from 'i18n-calypso';
+import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
+import { MaybeLink } from './maybe-link';
+import type { Task, Expandable } from '@automattic/launchpad';
+
+const removeDuplicatedSlashes = ( url: string ) => url.replace( /(https?:\/\/)|(\/)+/g, '$1$2' );
+
+const getPluginInstallationPage = ( fromUrl: string ) => {
+	if ( fromUrl !== '' ) {
+		return removeDuplicatedSlashes(
+			`${ fromUrl }/wp-admin/plugin-install.php?s=%2522migrate%2520guru%2522&tab=search&type=term`
+		);
+	}
+
+	return 'https://wordpress.org/plugins/migrate-guru/';
+};
+
+const getMigrateGuruPageURL = ( siteURL: string ) =>
+	removeDuplicatedSlashes( `${ siteURL }/wp-admin/admin.php?page=migrateguru` );
+
+const recordInstructionsLinkClick = ( linkname: string ) => {
+	recordTracksEvent( 'calypso_site_migration_instructions_link_click', {
+		linkname,
+	} );
+};
+
+interface Options {
+	fromUrl: string;
+}
+
+interface Step {
+	task: Task;
+	expandable?: Expandable;
+}
+
+type Steps = Step[];
+
+export const useSteps = ( { fromUrl }: Options ): Steps => {
+	const translate = useTranslate();
+
+	const steps = [
+		{
+			title: translate( 'Install the Migrate Guru plugin' ),
+			content: (
+				<p>
+					{ translate(
+						'First you’ll need to install and activate the {{a}}Migrate Guru plugin{{/a}} on the site you want to migrate. Click next when you’re ready.',
```

Comment:
Just curious if this text is from the original migration instructions page, or if it's ever been through a copy review? Asking because I see some issues. For example, in this sentence I would expect `next` to be `'Next'`.

Reply:
Oh! Good that you mentioned this! It came from Figma, but it was reviewed in the last version and I didn't notice it.

Updated in https://github.com/Automattic/wp-calypso/pull/92070/commits/2f8feaeff538900af5bb79b897860f9c34f6d458.

And based on your suggestion, I updated the "next" references to **Next** (with bold) to be like the reference to the **Migrate** button.

cc @fditrapani to take a look and confirm if it went through a copy review already.

Reply:
Good catch @donnapep. I'm not sure if it's been through copy review but will check!

Reply:
I connected with Jana and she shared this doc with me: https://docs.google.com/document/d/1af2XBfOK7wOdKt_oSii_h03SGFTbXxHQfEvu4nseFHQ/edit

I think we caught most of the issues. @renatho let me know if there are any discrepancies you want to go over.